docs: suggestions for updates to cluster docs (#4253)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/clusters/attach-cluster/cluster-with-network-restrictions/index.md

<p class="message--note"><strong>NOTE: </strong>If your cluster blocks public access, you may need to make the additional step of allowing certain authorized networks where Docker images are hosted for Konvoy to use your cluster, specifically <code>https://registry-1.docker.io/</code>.</p>

1. Select a workspace from the **Workspace Selector** in the top navigation bar. 

1. On the Dashboard page, select the **Add Cluster** option in the **Actions** dropdown menu at the top right.

1. Select **Attach Cluster**.

1. Select the **Cluster has networking restrictions** card to display the configuration page.

    <!--- ![Add Cluster Networking Options](/dkp/kommander/1.4/img/cluster-has-networking-restrictions.png) --->

1. Enter the **Cluster Name** of the cluster you're attaching.

1. Create additional new Labels as needed.

> 1. Select the hostname that is Ingress for the cluster from the **Load Balancer Hostname** dropdown menu. You will want the hostname to match the Kommander Host cluster that you are attaching your existing cluster with network restrictions to.

Reviewer: ```suggestion
1. Select the hostname that is the Ingress for the cluster from the **Load Balancer Hostname** dropdown menu. The hostname must match the Kommander Host cluster to which you are attaching your existing cluster with network restrictions.
```
